### ISSUE TITLE
Change Kept x profiles to Kept x cycles

### ISSUE THREAD

**Issue by j-harbin** [ArgoCanada/argoFloats#412]
Whenever we subset, we have a message that tells the user how many "profiles" were kept, see reprex below. It actually isn't the profiles, but rather the "cycles" (remember some cycles can have multiple profiles).

``` r
library(argoFloats)
ai <- getIndex()
index1 <- subset(ai, time=list(from="2021-02-15", to="2021-02-25"))
#> Kept 4440 profiles (0.184%)
```

<sup>Created on 2021-03-03 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by dankelley** [comment]
I figure we may as well clean this up before paper resubmission.  Did it fall under the radar, @j-harbin?

**Comment by j-harbin** [comment]
You're absolutely right. It did fall under the radar. It's now fixed in commit 3b1481a3cc8cf6d981c1dd489edcd14e42c80f06  of the develop branch.
